src/components/Cart/TotalCartProduct.tsx

```diff
@@ -26,10 +26,11 @@ const TotalCartProduct = () => {
         </p>
       </TotalProductCountWrapper>
       <Button
-        type='button'
-        primary={false}
-        size='small'
-        border
+        type="button"
+        buttonPrimary={false}
+        size="small"
+        buttonBorder
+        disabled={checkedCount === 0 ? true : false}
```

Comment:
아래처럼 축약해서 적어도 괜찮을거 같네용

```suggestion
        disabled={checkedCount === 0}
```
```suggestion
        disabled={!checkedCount}
```

Reply:
전체적으로 축약해서 수정해봤습니다! 놓친 부분이 많았네요 ㅎㅎ